### PR TITLE
updated search context with new vagrant environment

### PR DESCRIPTION
**Pull request by aheimsbakk**

### CODE REVIEW

File: doc/search_context.adoc [hunk 1]
```diff
@@ -5,15 +5,17 @@
...
+. link:http://10.10.10.10/pycsw/csw.py?mode=opensearch&service=CSW&version=2.0.2&request=GetCapabilities
 
 This will return a description document of the catalog service. The `URL` field in the description document is a template format that can be used to represent a parameterized form of the search. The search client will process the URL template and attempt to replace each instance of a template parameter, generally represented in the form {name}, with a value determined at query time (link:https://github.com/dewitt/opensearch/blob/master/opensearch-1-1-draft-6.md#opensearch-url-template-syntax[OpenSearch URL template syntax]). The question mark following any search parameter means that the parameter is optional.
 
 ===== Online catalog
 
-For searching the online metadata catalog, the base url (`http://10.20.30.10/`) must be replaced by `https://csw.s-enda.k8s.met.no/`.
+. link:http://10.10.10.10/?mode=opensearch&service=CSW&version=2.0.2&request=GetRecords&elementsetname=full&typenames=csw:Record&resulttype=results
+
+For searching the online metadata catalog, the base url (`http://10.10.10.10/`) must be replaced by `https://csw.s-enda.k8s.met.no/`.
```
Comment (mortenwh) [inline doc/search_context.adoc:18]:
```suggestion
For searching the online metadata catalog, the base url (`http://10.10.10.10/`) must be replaced by `https://csw.s-enda.k8s.met.no/`:
* `http://csw.s-enda.k8s.met.no/?mode=opensearch&service=CSW&version=2.0.2&request=GetRecords&elementsetname=full&typenames=csw:Record&resulttype=results`

```

**Review by mortenwh — CHANGES_REQUESTED** [review]
A few minor changes. I'll also push a change for the xml examples.

File: doc/search_context.adoc [hunk 2]
```diff
@@ -53,9 +55,9 @@ PyCSW opensearch only supports geographical searches querying for a box. For mor
         <csw:Constraint version="1.1.0">
             <ogc:Filter>
                 <ogc:Contains>
-		            <ogc:PropertyName>ows:BoundingBox</ogc:PropertyName>
-			        <gml:Point>
-                	        <gml:pos srsDimension="2">59.0 4.0</gml:pos>
+                <ogc:PropertyName>ows:BoundingBox</ogc:PropertyName>
```
[the remark this answers is not in the document]
Comment (aheimsbakk) [reply doc/search_context.adoc]: Just tell me when :)

[the remark this answers is not in the document]
Comment (mortenwh) [reply doc/search_context.adoc]: it's done in 4aa86d0  :)

**Review by mortenwh — APPROVED** [review]

**Pull request merged**